lib/icon-button/index.tsx

```diff
@@ -10,15 +10,22 @@ type OwnProps = {
 type Props = OwnProps;
 
 export const IconButton = ({ icon, title, ...props }: Props) => (
-  <Tooltip
-    classes={{ tooltip: 'icon-button__tooltip' }}
-    enterDelay={200}
-    title={title}
-  >
-    <button className="icon-button" type="button" data-title={title} {...props}>
-      {icon}
-    </button>
-  </Tooltip>
+  <span>
+    <Tooltip
+      classes={{ tooltip: 'icon-button__tooltip' }}
+      enterDelay={200}
+      title={title}
+    >
```

Comment:
did you try wrapping the `<button>` from inside the `<Tooltip>`?

I interpreted the error as a wrapper from inside `<Tooltip>` vs. wrapping `<Tooltip>`

Reply:
I thought I tried that first. Lemme see.

Reply:
Oh yeah. I tried that and it leads to the tooltip being misaligned:

<img width="671" alt="Screen Shot 2020-09-03 at 2 23 06 PM" src="https://user-images.githubusercontent.com/52152/92174153-08dfb000-edf1-11ea-882f-1efc536aeaa4.png">

Reply:
@dmsnell Okay I figured it out! https://github.com/Automattic/simplenote-electron/pull/2324